gulp/clean.js

```diff
@@ -18,7 +18,9 @@ export function cleanCSS() {
 	const delPath = [
 		`${paths.styles.dest}/**/*.css`,
 		`!${paths.styles.srcDir}`,
-		`!${paths.styles.srcDir}/**`
+		`!${paths.styles.srcDir}/**`,
```

Comment:
@benoitchantre I think a better way to do this would be to exclude all paths in `filesToCopy` defined in the config from the `clean` task rather than hard-coding the excludes.

Reply:
Yes, thank you for the review. I’ll change that.